tcl/org.testeditor.tcl.dsl.tests/src/org/testeditor/tcl/dsl/tests/formatter/AbstractTclFormatterTest.xtend

```diff
@@ -0,0 +1,13 @@
+package org.testeditor.tcl.dsl.tests.formatter
+
+import javax.inject.Inject
+import org.testeditor.tcl.dsl.tests.AbstractTclTest
+import com.google.inject.Provider
+import org.eclipse.xtext.junit4.formatter.FormatterTestRequest
+import org.eclipse.xtext.junit4.formatter.FormatterTester
+
+class AbstractTclFormatterTest extends AbstractTclTest {
+	@Inject Provider<FormatterTestRequest> formatterRequestProvider
+	@Inject protected FormatterTester formatterTester
```

Comment:
I would register this as an extension provider, i.e. `protected extension FormatterTester` and leave out the variable name in subclasses. This has the advantage that we could override `assertFormatted` in the future without touching all subclasses. Had this problem in another project (i.e. adjust the formatting request).